commands/env.go

```diff
@@ -210,5 +210,5 @@ func generateUsageHint(appName, machineName, userShell string) string {
 		}
 	}
 
-	return fmt.Sprintf("# Run this command to configure your shell: %s\n", cmd)
+	return fmt.Sprintf("# Run this command to configure your shell: \n  %s\n", cmd)
```

Comment:
I guess `eval` deals with it OK, but wouldn't it be safer to have a `#` at the beginning of the second line of the message?